react/react-training/src/app/screens/Dashboard/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Switch, Route } from 'react-router';
+import { ROUTES } from '@constants/routes.js';
+
+import Game from './screens/Game';
+import Logout from './screens/Logout';
+import Profile from './screens/Profile';
+import TopBar from './screens/Game/components/TopBar';
+
+function Dashboard() {
+  return (
+    <Router>
+      <Fragment>
```

Comment:
This `Fragment` is required?

Reply:
@henryzarza yesss... Si no lo tengo me sale este error:
`A <Router> may have only one child element` 